lib/linguist/languages.yml

```diff
@@ -5447,6 +5447,7 @@ Protocol Buffer Text Format:
   - ".textproto"
   - ".pbt"
   - ".pbtxt"
+  - ".txtpb"
```

Comment:
> The exception is the primary extension: it should always be first.

The canonical and preferred extension is `.txtpb` (see PR message), but the _most common_ extension appears to be `.textproto`. Which should be considered the "primary" extension and thus placed first?